samples/openthread/common/overlay-ot-defaults.conf

```diff
@@ -63,3 +63,4 @@ CONFIG_MBEDTLS_DHM_C=n
 CONFIG_MBEDTLS_RSA_C=n
 CONFIG_MBEDTLS_SHA1_C=n
 CONFIG_MBEDTLS_SHA512_C=n
+CONFIG_MBEDTLS_HEAP_SIZE=10240
```

Comment:
Did you try commissioning with oberon?
If nothing has changed this will not work. For some reason oberon required huge heap (it is strange, considering my last discussion with Frank about Oberon doing everything on stack), however this was not verified on latest oberon.